### DISCUSSION TITLE
Resume node?

### DISCUSSION THREAD

**Discussion by grazaerd** [chaiNNer-org/chaiNNer#2233] [Q&A] [answered]
Is there a resume node where you want to continue from previous task?

**Comment by joeyballentine** [comment]
Could you elaborate a bit more on how that would work? What exactly would you be resuming and how would that differ from the existing pause & resume functionality?

**Reply by grazaerd** [reply]
Sorry if my question was vague, but what I meant was to continue the previous task when I shut down Chainner or the computer. Example Scenario, 01.jpg, 02.jpg... and so on; afterwards, I stop at 50.jpg and want to continue tomorrow (starts at 51.jpg).

**Reply by joeyballentine** [reply, marked as answer]
An individual node isn't going to help you there. That's an overall feature that I think will be possible once I rewrite how iterators work
